studio/schemas/documents/admin/jobPostings.ts

```diff
@@ -0,0 +1,21 @@
+import { defineType } from "sanity";
+
+import { jobPostingID } from "studio/schemas/objects/jobPosting";
+
+export const jobPostingsID = "jobPostings";
+
+const jobPostings = defineType({
+  name: jobPostingsID,
+  type: "document",
+  title: "Job Postings",
+  fields: [
+    {
+      name: "jobPostings",
```

Comment:
Name should be unique, the name of the field is now the same as the name of the document :)

Reply:
Fixed in https://github.com/varianter/variant.no/pull/908/commits/c29246e8d078be00cf34f2504f4464f5920e404c